spec/passaporte_web/notification_spec.rb

```diff
@@ -156,13 +156,13 @@
         it "should return false an do nothing if the Notification is already persisted" do
           PassaporteWeb.configuration.user_token = "f01d30c0a2e878fecc838735560253f9e9395932f5337f40"
           notification = described_class.find_all(1,20,nil,true).notifications.last
-          notification.save.should be_false
+          expect(notification.save).to be_falsy
         end
         it "should return false and set the errors with the reason for the failure, authenticated as the user" do
           PassaporteWeb.configuration.user_token = "f01d30c0a2e878fecc838735560253f9e9395932f5337f40"
           notification.target_url = 'lalalala'
           notification.should_not be_persisted
-          notification.save('user').should be_false
+          expect(notification.save('user')).to be_falsy
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.